packages/edit-site/src/components/navigation-sidebar/navigation-panel/style.scss

```
@@ -87,6 +87,17 @@
 .edit-site-navigation-panel__template-item {
 	display: block;
 
+	&.is-active {
+		.edit-site-navigation-panel__template-item-description {
+			color: $gray-100;
+		}
+
+		.edit-site-navigation-panel__template-item-customized-dot {
+			background: #fff;
+		}
+	}
+
+	button {
 	.components-button {
 		display: flex;
 		flex-direction: column;
```

```suggestion
	.components-button {
		display: flex;
		flex-direction: column;
```
It looks like this might have gotten messed up on the master rebase, and its causing it to fail the build.  The `button` is missing a closing brace.  We could either remove `button` now since we are using the selector for `.components-button`, or add the closing brace further below.

---

Actually seems the nesting ruins the styles, so best just remove the `button {` anyways.